tests/test_mongodb_log_based_interruptible.py

```diff
@@ -97,6 +97,7 @@ def get_properties(self):
                 'database': os.getenv('TAP_MONGODB_DBNAME')
                 }
 
+    @unittest.skip("Test is unstable")
```

Comment:
Can you link the backlog ticket, and remove the ipdb trace below

Reply:
done